custom_components/elegoo_printer/elegoo_sdcp/models/status.py

```
@@ -1,11 +1,30 @@
 
+class CurrentFanSpeed:
+    """Represents the speed of the various fans."""
+
+    def __init__(self, data: dict[str, Any] = {}) -> None:  # noqa: B006
+        """Initialize a new CurrentFanSpeed object."""
+        self.model_fan: int = data.get("ModelFan", 0)
+        self.auxiliary_fan: int = data.get("AuxiliaryFan", 0)
+        self.box_fan: int = data.get("BoxFan", 0)
+
+
+class LightStatus:
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)